sdk/core/Azure.Core/api/Azure.Core.net6.0.cs

```diff
@@ -381,6 +381,7 @@ public abstract partial class ClientOptions
         protected ClientOptions(Azure.Core.DiagnosticsOptions? diagnostics) { }
         public static Azure.Core.ClientOptions Default { get { throw null; } }
         public Azure.Core.DiagnosticsOptions Diagnostics { get { throw null; } }
+        public Azure.Core.RedirectPolicyOptions? RedirectPolicyOptions { get { throw null; } set { } }
```

Comment:
The other properties don't end in "Options". What about calling it ClientRedirects?